src/api/lib/middlewares/tokenChecker.js

```diff
@@ -17,6 +17,11 @@ export default async (ctx, next) => {
 
     try {
         ctx.user = await jwt.verify(token, config.apps.api.security.jwt.privateKey);
```

Comment:
We, then, you loose the ability to check expiration on client side (which is bad). We have to understand that we are using and need a tokenData object and not a user object, as a user object is not enough